modules/claims_api/spec/requests/v0/power_of_attorney_request_spec.rb

```diff
@@ -49,15 +49,6 @@
         expect(parsed['data']['type']).to eq('claims_api_power_of_attorneys')
         expect(parsed['data']['attributes']['status']).to eq('pending')
       end
-
-      it 'returns the same successful response with all the data' do
```

Comment:
this test (and the same test in v1) is expected to fail now, since a _new_ record will get created. should I add a different test in its place (like the scenario described in the description) or not worry about it?

Reply:
i think this is fine to just remove personally